util/emojis.py

```diff
@@ -0,0 +1,64 @@
+'''
+Utility functions for handling of emoji synchronization across guilds
+'''
+
+from tkinter import Image
+import discord
+from util.log import Log
+
+async def sync_add(bot: discord.Bot, emoji: discord.Emoji):
+    for guild in bot.guilds:
+        # Check if this emoji already exists
+        guild_emojis = await guild.fetch_emojis()
+        exists = False
+        for guild_emoji in guild_emojis:
+            if guild_emoji.name == emoji.name:
+                exists = True
+                break
+        
+        # If the emoji exists, we can skip creation
+        if exists:
+            continue
+
+        # Create emoji
+        try:
+            guild.create_custom_emoji(name=emoji.name, image=await emoji.read())
+            Log.ok(f'Emoji: {emoji.name} successfully added in {guild.name}')
+        except:
+            Log.warning(f'Could not create emoji {emoji.name} in {guild.name}')
+            continue
+
+async def sync_delete(bot: discord.Bot, emoji: discord.Emoji):
+    for guild in bot.guilds:
+        guild_emojis = guild.fetch_emojis()
```

Comment:
I think `fetch_emojis()` is `async`/must be `await`ed, almost all fetch functions are.